test/liveman/user/users_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Liveman.User.UsersTest do
+  use Liveman.DataCase, async: true
+
+  alias Liveman.User.Users
+
+  describe "register_user/1" do
```

Comment:
The tests duplicates the schema tests. Here there could be two tests: 1) test with valid params and 2) test it returns Ecto changeset error with valid params/

Reply:
Updated, used only valid and invalid param test